modin/config/pubsub.py

```diff
@@ -135,11 +135,14 @@ class Parameter(object):
         ``ValueSource``.
     """
 
-    choices: typing.Sequence[str] = None
+    choices: Optional[Sequence[str]] = None
     type = str
-    default = None
+    default: Optional[Any] = None
     is_abstract = True
-    _value_source = None
+    _value_source: Optional[int] = None
```

Comment:
ideally it should be an enum I think

Reply:
Technically yes. However I think `mypy` still treats it as an int, so it complains when we try to assign enum values to `_value_source` when it is defined as `Optional[ValueSource]`

Reply:
Or an [`IntEnum`](https://docs.python.org/3/library/enum.html#enum.IntEnum) if you still want it to be an int value.

Reply:
can we get this done as part of this PR?

Reply:
@vnlitvinov yes I can!